app/test_main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+from unittest import mock
+from app.main import can_access_google_page
+
+
+def test_called_function() -> None:
+    with (mock.patch("app.main.valid_google_url") as mock_valid,
+          mock.patch("app.main.has_internet_connection") as mock_internet):
```

Comment:
Use `mock.path` as decorator instead of context manager. It makes your code clearer.